src/modules/oauth/containers/ServerInteractionBox.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux';
+import { withRouter } from "react-router";
+import {
+  Alert, Button, Card, CardBody, Col, ListGroup, ListGroupItem, Label,
+  Form, FormGroup, Input, InputGroup, InputGroupAddon, InputGroupText, Row
+} from 'reactstrap';
+import axios from 'axios'
+
+class ServerInteractionBox extends Component {
```

Comment:
What is "server interaction"?

Reply:
This won't make a sense after refactorization is finished. I will remove it.